app.py

```diff
@@ -67,6 +67,7 @@
 import invesalius.utils as utils
 
 from invesalius import inv_paths
+from invesalius.net.pedal_connection import PedalConnection
```

Comment:
Move this import to the part you use PedalConnection

Reply:
Done.